src/lesson3/ChangeNumber.java

```diff
@@ -0,0 +1,19 @@
+package lesson3;
+
+import java.util.Scanner;
+
+public class ChangeNumber {
+      public static void main(String[] args) {
+          Scanner in = new Scanner(System.in);
+          long octal_num, decimal_num = 0;
```

Comment:
в джава використовують камел кейс
```suggestion
          long octalNum, decimalNum = 0;
```